tools/nni_cmd/nnictl.py

```diff
@@ -149,6 +141,28 @@ def parse_args():
     parser_trial_export.add_argument('--type', '-t', choices=['json', 'csv'], required=True, dest='type', help='target file type')
     parser_trial_export.add_argument('--filename', '-f', required=True, dest='path', help='target file path')
     parser_trial_export.set_defaults(func=export_trials_data)
+    #save experiment data
+    parser_save_data = parser_experiment_subparsers.add_parser('save', help='save experiment data')
+    parser_save_data.add_argument('id', nargs='?', help='the id of experiment')
+    parser_save_data.add_argument('--path', '-p', required=False, help='the folder path to store nni experiment data, \
+                                   default current working directory')
+    parser_save_data.add_argument('--saveCodeDir', '-s', action='store_true', default=False, help='save codeDir data \
+                                   of the experiment')
+    parser_save_data.set_defaults(func=save_experiment)
+    #open experiment data
+    parser_open_data = parser_experiment_subparsers.add_parser('load', help='load experiment data')
+    parser_open_data.add_argument('--path', '-p', required=True, help='the path of nni package file')
```

Comment:
better to make it consistent with the doc

Reply:
updated